INSTALLING_PACKAGES.md

```diff
@@ -1,7 +1,26 @@
 # Repository installation
 
-To install only virtualization related packages, for example, you can
-install the open-power-host-os-virt metapackage:
+To install Host OS packages from a repository, you'll first need a
+system with a
+[supported GNU/Linux distribution](README.md#supported-gnulinux-distributions)
+and a yum configuration file pointing to an accessible yum repository
+containing the Host OS packages.
+
+If you've just executed the build scripts and want to use the local
+repository, your yum configuration file will be available at
```

Comment:
build scripts => build script (there is only one)
local repository => built local YUM repository